### PR TITLE
0.11.0 release candidate 1

### PR DESCRIPTION
**Pull request by SWilson4**
Please see the associated pre-release: https://github.com/open-quantum-safe/liboqs/releases/tag/0.11.0-rc1

### CODE REVIEW

**Review by baentsch — COMMENTED** [review]
OK, I'm not asked for review, but there's at least one I have to make: Shouldn't this trigger _all_ CI?

**Comment by SWilson4** [comment]
> OK, I'm not asked for review, but there's at least one comment I have to make: Shouldn't this trigger _all_ CI?

You certainly should have been; I simply missed adding reviewers. :)

It triggered the extended and oqs-provider release tests as well as all of the standard (PR) tests. The former don't show up here since they were triggered by the release event and not by the pull_request event.

Here's the liboqs run:
https://github.com/open-quantum-safe/liboqs/actions/runs/10857298451

and here's the downstream run in provider:
https://github.com/open-quantum-safe/oqs-provider/actions/runs/10857300499

**Review by baentsch — APPROVED** [review]
LGTM. But is this meant to land or only when final?

**Comment by beldmit** [comment]
Do you plan to land the final version of ML-DSA later or the current version is already the final one?

**Comment by baentsch** [comment]
> Do you plan to land the final version of ML-DSA later or the current version is already the final one? 

Later on, see #1919. The current version is still 'ipd'.

**Comment by beldmit** [comment]
Great! Do you plan to land it in 0.11 or later?

**Comment by baentsch** [comment]
Decision was to push it to 0.12.0.

**Comment by beldmit** [comment]
Thanks for clarification!

File: RELEASE.md [hunk 1]
```diff
@@ -14,117 +14,118 @@ liboqs can be used with the following Open Quantum Safe application integrations
...
+This release updates ML-KEM implementations to their [final FIPS 203](https://csrc.nist.gov/pubs/fips/203/final) versions (OQS continues to support NIST Round 3 version of Kyber for interoperability purposes). Additionally, this release adds support for MAYO and CROSS digital signature schemes from [NIST Additional Signatures Round 1](https://csrc.nist.gov/Projects/pqc-dig-sig/round-1-additional-signatures) along with stateful hash-based signature schemes [XMSS](https://datatracker.ietf.org/doc/html/rfc8391) and [LMS](https://datatracker.ietf.org/doc/html/rfc8554). Finally, this release provides formally verified implementations of Kyber-512 and Kyber-768 from [libjade](https://github.com/formosa-crypto/libjade/releases/tag/release%2F2023.05-2).
```
Comment (dstebila) [inline RELEASE.md:31]:
```suggestion
This release updates ML-KEM implementations to their [final FIPS 203](https://csrc.nist.gov/pubs/fips/203/final) versions. This release still includes the NIST Round 3 version of Kyber for interoperability purposes, but we plan to remove Kyber Round 3 after liboqs 0.11. Additionally, this release adds support for MAYO and CROSS digital signature schemes from [NIST Additional Signatures Round 1](https://csrc.nist.gov/Projects/pqc-dig-sig/round-1-additional-signatures) along with stateful hash-based signature schemes [XMSS](https://datatracker.ietf.org/doc/html/rfc8391) and [LMS](https://datatracker.ietf.org/doc/html/rfc8554). Finally, this release provides formally verified implementations of Kyber-512 and Kyber-768 from [libjade](https://github.com/formosa-crypto/libjade/releases/tag/release%2F2023.05-2).
```

**Review by dstebila — APPROVED** [review]

**Review by dstebila — CHANGES_REQUESTED** [review]
Looks good! One suggestion for wording around Kyber Round 3 but otherwise good to go.

**Comment by SWilson4** [comment]
> LGTM. But is this meant to land or only when final?

Only when final.

> Looks good! One suggestion for wording around Kyber Round 3 but otherwise good to go.

Perfect, I'll make the change in the release notes when updating the version string to remove "-rc1".

**Review by dstebila — APPROVED** [review]

**Comment by baentsch** [comment]
As discussed yesterday (to give feedback on testing intentions) I don't see a good reason to test this with oqsprovider until we have re-established functional parity with CF again, see also  https://github.com/openssl/project/issues/844. Realistically I don't see a risk there though, just need to wait a bit until @bhess is back from ICMC. Any other thoughts, @SWilson4 @praveksharma (also as I'm looking to you for doing the actual oqsprovider release as well as any possible further downstream releases)?

**Comment by SWilson4** [comment]
In an in-person meeting with @dstebila and @praveksharma earlier this week, we agreed to hold 0.11.0 until https://github.com/open-quantum-safe/oqs-provider/issues/503 / https://github.com/open-quantum-safe/oqs-provider/pull/524 are resolved / tested to enable thorough testing of ML-KEM.

**Comment by SWilson4** [comment]
Closing as #1939 has landed.

**Pull request closed without merging**